src/examples/java/com/amazonaws/crypto/examples/EscrowedEncryptExample.java

```diff
@@ -66,23 +71,25 @@ public static void main(final String[] args) throws Exception {
     }
 
     private static void standardEncrypt(final String kmsArn, final String fileName) throws Exception {
-        // Standard user encrypting to both KMS and the escrow public key
+        // Standard practice: encrypt with the KMS CMK and the escrowed public key
```

Comment:
I'm hesitant to call this "standard practice", given that it's just an example of how you might do something in one very specific way.  It's just a path which uses all available master keys.